### ISSUE TITLE
Specifying a table width less than the combined column widths breaks

### ISSUE THREAD

**Issue by avand** [tj/terminal-table#23]
I set the width of a table to less than the width of all the columns combined and got this error:

```
undefined local variable or method `wanted' for #<Terminal::Table:0x007fd99a490248>
/Users/avand/Code/sqoot/api/vendor/ruby/1.9.1/gems/terminal-table-1.4.5/lib/terminal-table/table.rb:169:in `additional_column_widths'
/Users/avand/Code/sqoot/api/vendor/ruby/1.9.1/gems/terminal-table-1.4.5/lib/terminal-table/table.rb:85:in `column_width'
/Users/avand/Code/sqoot/api/vendor/ruby/1.9.1/gems/terminal-table-1.4.5/lib/terminal-table/separator.rb:7:in `block in render'
/Users/avand/Code/sqoot/api/vendor/ruby/1.9.1/gems/terminal-table-1.4.5/lib/terminal-table/separator.rb:6:in `map'
/Users/avand/Code/sqoot/api/vendor/ruby/1.9.1/gems/terminal-table-1.4.5/lib/terminal-table/separator.rb:6:in `render'
/Users/avand/Code/sqoot/api/vendor/ruby/1.9.1/gems/terminal-table-1.4.5/lib/terminal-table/table.rb:120:in `block in render'
/Users/avand/Code/sqoot/api/vendor/ruby/1.9.1/gems/terminal-table-1.4.5/lib/terminal-table/table.rb:120:in `map'
/Users/avand/Code/sqoot/api/vendor/ruby/1.9.1/gems/terminal-table-1.4.5/lib/terminal-table/table.rb:120:in `render'
```

Looks like the exception references a non-existent variable. Should be a pretty easy fix.

**Comment by nateberkopec** [comment]
Fixed in #30

**Comment by avand** [comment]
👍
